src/static/css/nav-primary.less

```diff
@@ -66,398 +66,320 @@
     - cfgov-nav-primary
 */
 
-.primary-nav .list_link {
-    .u-link__colors(@black, @green);
-    .webfont-regular();
-}
+.primary-nav {
+    .list_link {
+        .webfont-regular();
+    }
 
-.sub-nav_title {
-    .webfont-regular();
-    font-size: 1em;
-    color: @pacific;
-}
+    &_trigger {
+        width: 60px;
+        height: 60px;
 
-/* Sliding nav for portrait tablets and smaller
-   ========================================================================== */
+        display: none;
+        float: left;
 
-.respond-to-max(@tablet-max, {
+        background: @white;
+        border: none;
+        border-right: 1px solid @gray-50;
+        color: @gray;
+        font-size: 1.5em;
+
+        &:focus {
+            outline: 1px dotted @black;
+        }
+
+        &:before {
+            .cf-icon();
+
+            content: @cf-icon-menu;
+            vertical-align: middle;
+        }
+
+        &[aria-expanded="true"] {
```

Comment:
@Scotchester I agree, and thought it was documented somewhere, but now I can't find it.